test/xspec-xmlns.sch

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sch:schema xmlns:my="http://example.org/ns/my" xmlns:d="http://example.org/ns/default"
+   xmlns:dd="http://example.org/ns/default/description"
```

Comment:
I think that in typical use cases, these `xmlns:my`, `xmlns:d` and `xmlns:dd` are removed and they're declared only by `sch:ns`.

Reply:
Great, thanks for the tip. The removals will be in my next upload (sometime in the next few days).

Reply:
Resolved in 40b65ac37638ce4c2795558a670d874ccb1551bf